frontend/app/admin/src/views/settings/membership-types/components/PlatformMembershipTypeReduceablePriceEdit.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <ReduceablePriceInput v-model="model" :title="$priceTitle" :validator="validator" :error-box="errorBox">
+        <template #end>
+            <slot />
+        </template>
+    </ReduceablePriceInput>
+</template>
+
+<script setup lang="ts">
+import { ErrorBox, ReduceablePriceInput, usePlatform, Validator } from '@stamhoofd/components';
+import { ReduceablePrice } from '@stamhoofd/structures';
+import { computed } from 'vue';
+
+const props = defineProps<{
+    tagId: string,
+    showPricePerDay: boolean,
+    validator: Validator,
+    errorBox?: ErrorBox | null
+}>();
+
+const model = defineModel<ReduceablePrice>({ required: true });
+// const {patched, hasChanges, addPatch, patch} = usePatch(props.reduceablePrice);
+
+const $platform = usePlatform();
+const tag = $platform.value.config.tags.find(t => t.id === props.tagId);
+// const $reduceablePrice = ref(props.reduceablePrice);
+const $priceTitle = computed(() => format(props.showPricePerDay ? 'Vaste prijs' : 'Prijs'));
```

Comment:
Onduidelijk waarom je soms $ in de naam van variabelen gebruikt

Reply:
Globale variabelen hebben vaak $ in de naam om conflicten en duplicates te vermijden, maar voor lokale is dat niet nodig